core/domain/user_services_test.py

```diff
@@ -132,7 +132,9 @@ def test_set_invalid_usernames(self):
         user_id = 'someUser'
         user_services.create_new_user(user_id, 'user@example.com')
         bad_usernames = [
-            ' bob ', '@', '', 'a' * 100, 'ADMIN', 'admin', 'AdMiN2020']
+            ' bob ', '@', '', 'a' * 100, 'ADMIN', 'admin', 'AdMiN2020',
```

Comment:
Add some that have "oppia" but no "bot".

Reply:
Done!